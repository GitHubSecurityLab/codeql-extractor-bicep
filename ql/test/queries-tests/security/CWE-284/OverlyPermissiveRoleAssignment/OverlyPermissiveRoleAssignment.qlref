security/CWE-284/OverlyPermissiveRoleAssignment.ql