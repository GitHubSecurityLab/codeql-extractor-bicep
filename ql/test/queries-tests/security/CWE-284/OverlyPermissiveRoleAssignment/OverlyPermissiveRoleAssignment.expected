| app.bicep:16:1:24:2 | This role assignment grants excessive privileges: Contributor role assigned at subscription scope. Consider using a more restrictive role or narrowing the scope to specific resources. |
| app.bicep:27:1:35:2 | This role assignment grants excessive privileges: Owner role assigned at subscription scope. Consider using a more restrictive role or narrowing the scope to specific resources. |
| app.bicep:38:1:46:2 | This role assignment grants excessive privileges: Contributor role assigned at resource group scope. Consider using a more restrictive role or narrowing the scope to specific resources. |
| app.bicep:49:1:57:2 | This role assignment grants excessive privileges: User Access Administrator role assigned at resource group scope. Consider using a more restrictive role or narrowing the scope to specific resources. |